app/flows/check_uk_visa_flow.rb

```diff
@@ -171,7 +171,8 @@ def define
           outcome :outcome_transit_refugee_not_leaving_airport
         elsif calculator.passport_country_in_direct_airside_transit_visa_list?
           outcome :outcome_transit_not_leaving_airport
-        elsif calculator.passport_country_in_visa_national_list? || calculator.travel_document?
+        elsif calculator.passport_country_in_visa_national_list? ||
```

Comment:
Looks like something about the lint style has changed? Do we know why?

Reply:
Also, do we have a tech task to clean up this chain of if-else? If not, think it would be a good idea to create one